master/buildbot/schedulers/base.py

```diff
@@ -151,6 +178,13 @@ def _changeCallback(self, key, msg, fileIsImportant, change_filter,
                     'not processed by scheduler %(name)s',
                     codebase=change.codebase, name=self.name)
             return
+
+        if not self.enabled:
```

Comment:
Shouldn't we rather stop the scheduler?
When it is restarted again it will consume changes that were left rather than just forget them